docs/cli/cli-subscriptions.md

```diff
@@ -4,4 +4,440 @@ title: CLI for Subscriptions API Tutorial
+            ],
+            [
+              140.1031494140625,
+              35.42374884923695
+            ],
+            [
+              140.1031494140625,
+              35.77102915686019
+            ],
+            [
+              139.5648193359375,
+              35.77102915686019
+            ],
+            [
+              139.5648193359375,
+              35.42374884923695
+            ]
+          ]
+        ],
+        "type": "Polygon"
+      },
+      "item_types": [
+        "PSScene"
+      ],
+      "start_time": "2023-03-17T04:08:00.0Z"
+    }
+  },
+  "tools": [
+    {
+      "type": "clip",
+      "parameters": {
+        "aoi": {
+          "coordinates": [
+            [
+              [
+                139.5648193359375,
+                35.42374884923695
+              ],
+              [
+                140.1031494140625,
+                35.42374884923695
+              ],
+              [
+                140.1031494140625,
+                35.77102915686019
+              ],
+              [
+                139.5648193359375,
+                35.77102915686019
+              ],
+              [
+                139.5648193359375,
+                35.42374884923695
+              ]
+            ]
+          ],
+          "type": "Polygon"
+        }
+      }
+    }
+  ],
+  "delivery": {
+    "type": "google_cloud_storage",
+    "parameters": {
+      "bucket": "pl-sub-bucket",
+      "credentials": "<REDACTED>"
+    }
+  }
+}
+```
+
+This is a full subscriptions JSON request, with the credentials redacted, so you'll have 
+to replace your own for it to work. Below we'll show the convenience methods that will 
+help create a custom one more easily, but if you'd just like to get things working then 
+just replace the 'delivery' section with your cloud credentials, see the 
+[core subscriptions delivery docs](https://developers.planet.com/docs/subscriptions/delivery/) 
+for more information. 
+
+To create a new subscription with the CLI you just use the `create` command:
+
+```
+planet orders subscriptions create my-subscription.json
+```
+
+The above command assumes that you've saved the subscriptions JSON as `my-subscription.json`.
+
+### List Subscriptions
+
+Now that you've got a subscription working you can make use of the other commands.
+
+```
+planet subscriptions list
+```
+
+Will output the JSON for your first 100 subscriptions. If you'd like more you can set the `--limit` 
+parameter higher, or you can set it to 0 and there will be no limit. 
+
+You can get nicer formatting with `--pretty` or pipe it into `jq`, just like the other Planet
+CLI's.
+
+The `list` command also supports filtering by the status of the subscription:
+
+```
+planet subscriptions list --status running
+```
+
+will give you just the currently active subscriptions. The other available status's are:
+`cancelled`, `preparing`, `pending`, `completed`, `suspended`, and `failed`.
+
+### Describe Subscription
```

Comment:
Note this section should change with #889 - just change the three places it says 'describe' with 'get' once that change is made.

Reply:
FYI: PR #900 has been merged to `main`, which closed this ticket in reference.